app/src/main/java/net/gsantner/opoc/util/GsFileUtils.java

```diff
@@ -752,4 +754,32 @@ public static List<File> replaceFilesWithCachedVariants(@Nullable List<File> fil
         }
         return files;
     }
+
+    // Get intent file
+    public static File getIntentFile(final Intent intent, final File fallback) {
```

Comment:
please have that one in ./app/src/main/java/net/gsantner/markor/util/MarkorContextUtils.java